model/generate.go

```diff
@@ -392,3 +398,21 @@ func (g *GeneratedProject) validateGeneratedProject(p *Project, cachedProject pr
 
 	return errors.WithStack(catcher.Resolve())
 }
+
+func updateMergeTaskDependencies(p *Project, v *Version) error {
+	execPairs, _, err := p.BuildProjectTVPairsWithAlias(evergreen.CommitQueueAlias)
+	if err != nil {
+		return errors.Wrap(err, "can't get alias pairs")
+	}
+
+	execTaskIDTable := NewTaskIdTable(p, v, "", "").ExecutionTasks
+	mergeTaskDependencies := make([]task.Dependency, 0, len(execPairs))
+	for _, pair := range execPairs {
+		depID := execTaskIDTable.GetId(pair.Variant, pair.TaskName)
+		mergeTaskDependencies = append(mergeTaskDependencies, task.Dependency{TaskId: depID, Status: evergreen.TaskSucceeded})
```

Comment:
I'd combine these lines, but I suspect it doesn't matter much.

Reply:
done.